### PR TITLE
ui/15399703-Page title

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

### CODE REVIEW

File: src/components/shared/Extensions/PageTitle/PageTitle.module.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.wrapper {
+  &::after {
+    @apply bg-gray-200 mt-6;
+
+    display: block;
+    height: 1px;
```
Comment (krawczykkatarzyna) [inline src/components/shared/Extensions/PageTitle/PageTitle.module.css:6]: please use rem

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import { PageTitleProps } from './types';
+import styles from './PageTitle.module.css';
+
+const displayName = 'PageTitle';
+
+const PageTitle = ({ title, subtitle }: PageTitleProps) => (
+  <div className={classNames(styles.wrapper)}>
+    <div className="flex gap-12 items-baseline">
+      <div className="max-w-[872px]">
```
Comment (krawczykkatarzyna) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:12]: the same

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        <h1 className="text-2xl text-gray-900 mb-4 font-semibold">{title}</h1>
+        <h2 className="text-xl text-gray-600 leading-normal">{subtitle}</h2>
+      </div>
+      <div className="hidden lg:block ">
+        {/* it's placeholder for sub navigation */}
+        <ul className="flex gap-8 font-semibold text-md text-gray-700 ">
+          <li>Pay</li>
```
Comment (krawczykkatarzyna) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:19]: please use intl and add translation

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+          <li>Decide</li>
```
Comment (krawczykkatarzyna) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:20]: the same

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+          <li>Manage</li>
```
Comment (krawczykkatarzyna) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:21]: the same

File: src/components/shared/Extensions/PageTitle/PageTitle.module.css (hunk 1 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/shared/Extensions/PageTitle/PageTitle.module.css:6]: `@apply bg-gray-200 mt-6 block w-full h-px content-none;`

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx (hunk 2 above)
Comment (KatarzynaStefanowiczPagepro) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:12]: Please, use rem

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+import styles from './PageTitle.module.css';
+
+const displayName = 'PageTitle';
+
+const PageTitle = ({ title, subtitle }: PageTitleProps) => (
+  <div className={classNames(styles.wrapper)}>
+    <div className="flex gap-12 items-baseline">
+      <div className="max-w-[872px]">
+        <h1 className="text-2xl text-gray-900 mb-4 font-semibold">{title}</h1>
+        <h2 className="text-xl text-gray-600 leading-normal">{subtitle}</h2>
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:14]: If `subtitle` is an optional value, please use:
`{subtitle && <h2 className="text-xl text-gray-600 leading-normal">{subtitle}</h2>}`

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+      </div>
+      <div className="hidden lg:block ">
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:16]: Remove unnecessary blank space

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+        {/* it's placeholder for sub navigation */}
+        <ul className="flex gap-8 font-semibold text-md text-gray-700 ">
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:18]: The same

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx [hunk 9]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { PageTitleProps } from './types';
+import styles from './PageTitle.module.css';
+
+const displayName = 'PageTitle';
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:7]: const displayName = 'Extensions.PageTitle';

File: tsconfig.base.json [hunk 10]
```diff
@@ -1,104 +1,51 @@
...
-      "node_modules/@types"
-    ],
-    "types": [
-      "node",
-      "jest"
-    ]
-  }
-}
\ No newline at end of file
+    "compilerOptions": {
```
Comment (krawczykkatarzyna) [inline tsconfig.base.json:2]: why it changed?

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+
+const PageTitle = ({ title, subtitle }: PageTitleProps) => (
+  <div className={classNames(styles.wrapper)}>
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:10]: ` <div className={styles.wrapper}>`

File: src/stories/common/PageTitle.stories.tsx [hunk 12]
```diff
@@ -0,0 +1,31 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import PageTitle from '~shared/Extensions/PageTitle';
+
+const meta: Meta<typeof PageTitle> = {
+  title: 'Shared/Page Title',
```
Comment (krawczykkatarzyna) [inline src/stories/common/PageTitle.stories.tsx:5]: title: 'Common/Page Title',

File: src/components/shared/Extensions/PageTitle/PageTitle.tsx [hunk 13]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+
+import { PageTitleProps } from './types';
+import styles from './PageTitle.module.css';
+
+const displayName = 'Extensions.PageTitle';
+
+const MSG = defineMessages({
```
Comment (krawczykkatarzyna) [inline src/components/shared/Extensions/PageTitle/PageTitle.tsx:9]: move it to const file or use to translation  const { formatMessage } = useIntl(); and move text to json with english translactions

**Review by KatarzynaStefanowiczPagepro — APPROVED** [review]

**Pull request merged**